Composer/packages/server/src/controllers/project.ts

```diff
@@ -518,51 +518,50 @@ function createProjectV2(req: Request, res: Response) {
     jobId: jobId,
   });
 }
+
 async function createProjectAsync(req: Request, jobId: string) {
-  let { templateId } = req.body;
-  const {
-    name,
-    description,
-    storageId,
-    location,
-    schemaUrl,
-    locale,
-    preserveRoot,
-    templateDir,
-    eTag,
-    alias,
-  } = req.body;
+  const { templateId } = req.body;
+
+  // todo: add back templateDir, eTag, alias from req extraction for PVA scenarios
```

Comment:
lets do this before merging

Reply:
I was originally planning on leaving PVA out in this PR (I am tracking its reintegration in #5498 ) but can look into it. It should be simple to put back in this v2 flow and test since it has its own isolated flow.

Reply:
Would this then break PVA in the nightly builds until fixed?

Reply:
It should not, there are no PVA unit tests running through this code. This PR should introduce a separate code path for creation that no existing production code paths or build blocking UTs run through unless you have the hidden feature flag checked.

Reply:
@benbrown Ive restored it and done initial testing with favorable results except there are still some quirks. Getting an env to test this e2e with PVA is blocking me to iron out these quirks efficiently. I have created a task to validate the PVA flows and Tony has offered to help which should expedite the PVA integration (I believe its just one more small change needed I just cant test the damn thing effectively).

Since this will not affect anything in prod, PVA included, is it okay to iron out the kinks in its own PR so we can unblock this PR? I should I work with Tony now to get this in during this PR?